client/my-sites/post-selector/pagination.jsx

```diff
@@ -0,0 +1,64 @@
+/**
```

Comment:
It's hiding the Query in a sub-component, which seems unnecessary at a first glance.